### DISCUSSION TITLE
distance transform

### DISCUSSION THREAD

**Discussion by niovl** [libvips/libvips#3957] [Q&A]
> Hi, @jcupitt 

 I wanted to use a method like [scipy.distance_transform_edt(input, sampling=None,)](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.distance_transform_edt.html) on very large size images (like 30000 x 50000), but I didn't find a corresponding one in libvips. Are there any plans to develop this in the future?

**Comment by jcupitt** [comment]
Hi @niovl,

`fill_nearest` might do what you want:

https://www.libvips.org/API/current/libvips-morphology.html#vips-fill-nearest

Though I don't know how well it would work with very large images. You'll probably need a lot of memory!
